### PR TITLE
BRITECH-146 changed speaker-comms references to comms

### PR DESCRIPTION
**Pull request by thullSL**

### CODE REVIEW

File: dataCenter/singleEcs.json [hunk 1]
```diff
@@ -16,11 +16,11 @@
         },
         "dynamoDbSecretAccessKey" : {
             "Type" : "String",
-            "Description" : "The secret access key with which speaker-comms talks to it's DB"
+            "Description" : "The secret access key with which comms talks to it's DB"
```
Comment (csscottlogic) [inline dataCenter/singleEcs.json]: "its"

**Comment by csscottlogic** [comment]
:shipit:

File: dataCenter/singleEcs.json [hunk 2]
```diff
@@ -16,11 +16,11 @@
...
         },
         "dynamoDbAccessKeyId" : {
             "Type" : "String",
-            "Description" : "The access key ID with which speaker-comms talks to it's DB"
```
Comment (jebbinSCL) [inline dataCenter/singleEcs.json]: Same as above "its"

**Comment by jebbinSCL** [comment]
:shipit: after typos

File: dataCenter/singleEcs.json (hunk 1 above)
Comment (thullSL) [inline dataCenter/singleEcs.json]: That's a possessive apostrophe
> That is the Speaker-Comm's Database

File: dataCenter/singleEcs.json (hunk 2 above)
Comment (thullSL) [inline dataCenter/singleEcs.json]: Sam as above "possessive"

File: dataCenter/singleEcs.json (hunk 1 above)
Comment (lclarksl) [inline dataCenter/singleEcs.json]: You don't use possessive apostrophes with it

Comment (thullSL) [inline dataCenter/singleEcs.json]: It's my prerogative

**Pull request merged**